src/main/java/uk/gov/hmcts/reform/prl/models/dto/ccd/BulkPrintOrderDetail.java

```diff
@@ -0,0 +1,23 @@
+package uk.gov.hmcts.reform.prl.models.dto.ccd;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.Getter;
+import lombok.Setter;
+
+@Data
+@Getter
+@Setter
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@Builder(toBuilder = true)
+@AllArgsConstructor
+public class BulkPrintOrderDetail {
+
+    private final String bulkPrintId;
+    private final String partyId;
+    private final String partyName;
+    private final String servedDateTime;
```

Comment:
Can we add served order details as well here? so it helps the user to know what all were included as part of the order pack

Reply:
Hi @shashisk97 Is it really needed ?
We are storing these details in each order served, so if we add served order details it will be duplicate details. 
Please confirm.